core/src/nodes/listeners.rs

```diff
@@ -74,6 +74,7 @@ use {Multiaddr, Transport};
 /// tokio::run(future.map_err(|_| ()));
 /// # }
 /// ```
+#[must_use = "streams do nothing unless polled"]
```

Comment:
As `ListenersStream` does not implement `Stream` it is better to refer to it directly, as in "ListenersStream does nothing unless polled".

Reply:
I stand corrected. It actually implements `Stream`.